src/python/WMCore/MicroService/MSManager.py

```diff
@@ -317,8 +318,12 @@ def info(self, reqName=None, **kwargs):
                 data['transferDoc'] = transferDoc[0]
 
         if 'unmerged' in self.services:
+            detail = strToBool(kwargs.get('detail', False))
+
             if kwargs.get("rse_type"):
```

Comment:
Since there are other comments in the PR, let me make the following as well.
It just ocurred to me that we could replace these 4 lines here by:
```
data = self.msUnmerged.getStatsFromMongoDB(detail=detail, rse_type=kwargs.get('rse_type'), rse=kwargs.get('rse'))
```

Reply:
And this changes the logic completely, by making all queries passed at once and hence no separation between the interfaces is achieved. Which was exactly the goal in the first place when I was calling the function to make the queries to the database once per API. This way we can maintain a decent set of well separated calls to the databases providing well defined results. While the other way around one would need to invent a really complicate call to the database to satisfy a REST call of the type: `rse=T2_CH_CERN&rse_type=t2&rse=T3_US_NERSC&detail=true`

BTW I already tried to mix the two calls `rse_type=...&rse=...` and actually the best result achieved was the values returned by the last query to the database to overwrite the others. So long sotrry short, mixing the APIs in a single calls from the top level method would complicate the call to the database itself quite a lot and would prevent us from keeping short and well understood filters, as it is rigt now.